frontend/webpack.circular-deps.ts

```diff
@@ -81,7 +92,7 @@ const applyThresholds = (
   compilation: webpack.compilation.Compilation,
 ) => {
   const totalCycles = cycles.length;
-  if (thresholds.totalCycles && totalCycles > thresholds.totalCycles) {
+  if (typeof thresholds.totalCycles === 'number' && totalCycles > thresholds.totalCycles) {
```

Comment:
Would this be simpler as 

```suggestion
  if (!thresholds.totalCycles || totalCycles > thresholds.totalCycles) {
```

Reply:
There are also default values for both thresholds so this is probably an unnecessary condition anyway lol